static/js/manage/manage-searches.js

```diff
@@ -0,0 +1,48 @@
+MEDUSA.manage.manageSearches = function() {
+    // Get total number current scene exceptions per source. Will request medusa, xem and anidb name exceptions.
+    var url = 'manage/searches/sceneexception/retrieve';
+    var updateExceptionTable = function(exceptions) {
+        var status = $('#sceneExceptionStatus');
+
+        var table = $('<ul></ul>').append('<li>' +
+            '<a href="' + MEDUSA.config.anonRedirect +
+            'https://github.com/pymedusa/Medusa/wiki/Scene-exceptions-and-numbering">' +
+            "Last updated medusa's exceptions</a> " +
+            new Date(exceptions.data.last_update.custom_exceptions * 1000)
+                .toLocaleDateString()).append('<li>' +
+            '<a href="' + MEDUSA.config.anonRedirect +
+            'http://thexem.de">' +
+            'Last updated xem exceptions</a> ' +
+            new Date(exceptions.data.last_update.xem * 1000)
+                .toLocaleDateString()).append('<li>Last updated anidb exceptions ' +
+            new Date(exceptions.data.last_update.anidb * 1000)
+                .toLocaleDateString());
```

Comment:
@OmgImAlexis i'm not sure how to format this. I'd personally start every .append on a new line.